tests/e2e/test_vdbench_workload.py

```diff
@@ -0,0 +1,279 @@
+"""
+Test to exercise VDBench Workload
+"""
+import logging
+import pytest
+import time
+import os
+
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.utility.utils import TimeoutSampler, ocsci_log_path
+from ocs_ci.ocs.resources.ocs import OCS
+from ocs_ci.utility import templating
+from ocs_ci.ocs.utils import get_pod_name_by_pattern
+from ocs_ci.ocs.ripsaw import RipSaw
+from ocs_ci.ocs import constants
+from ocs_ci.framework.testlib import E2ETest, workloads
+from tests import helpers
+
+log = logging.getLogger(__name__)
+
+
+# TODO: move this function to pod.py helper functions
+def download(pod_name, localpath, remotepath):
+    """
+    Download a file from pod
+
+    Args:
+        pod_name (str): Name of the pod
+        localpath (str): Local file to download
+        remotepath (str): Target path on the pod
+
+    """
+    cmd = f'cp {pod_name}:{remotepath} {os.path.expanduser(localpath)}'
+    ocp_obj = OCP()
+    ocp_obj.exec_oc_cmd(command=cmd)
+
+
+# TODO: move this function to helper.py helper functions
+def get_labeled_nodes(label):
+    """
+    Fetches all worker nodes.
+
+    Args:
+        label (str): node label to look for
+    Returns:
+        list: List of names of labeled nodes
+    """
+    ocp_node_obj = OCP(kind=constants.NODE)
+    nodes = ocp_node_obj.get(selector=label).get('items')
+    labeled_nodes_list = [node.get('metadata').get('name') for node in nodes]
+    return labeled_nodes_list
+
+
+def label_node(node, label):
```

Comment:
You can use `label_worker_node` in helpers.py

Reply:
The label_worker_node in helpers.py is not generic, it only set the label to OCS, if we will changed it to be more generic, it will do the  work.

Reply:
Sure, maybe this can be handled in a subsequent PR. Let's not hold this PR for making that change.

Reply:
i just fixed it with commit : 7775fe4